charts/gpu-metrics-exporter/templates/daemonset.yaml

```diff
@@ -5,68 +5,69 @@ metadata:
   namespace: {{ .Release.Namespace }}
   labels:
     {{- include "gpu-metrics-exporter.labels" . | nindent 4 }}
+  {{- if .Values.dcgmExporter.enabled }}
   annotations:
-    {{- if .Values.dcgmExporter.enabled }}
     ignore-check.kube-linter.io/privileged-container: "This daemon set needs to run DCGM Exporter as privileged to access the GPU metrics."
     ignore-check.kube-linter.io/run-as-non-root: "This daemon set needs to run DCGM Exporter as root to access the GPU metrics."
-    ignore-check.kube-linter.io/privilege-escalation-container: "This daemon set needs escalate privileges for DCGM Exporter to access the GPU metrics."
-    ignore-check.kube-linter.io/no-read-only-root-fs: "This daemon set needs to run DCGM Exporter with read-only root filesystem to access the GPU utilization metrics."
-    {{- end }}
+    ignore-check.kube-linter.io/privilege-escalation-container: "This daemon set needs escalate privileges for DCGM Exporter."
+    ignore-check.kube-linter.io/no-read-only-root-fs: "This daemon set needs to run DCGM Exporter with read-only root filesystem."
+    ignore-check.kube-linter.io/drop-net-raw-capability: "This daemon set needs to run DCGM Exporter with SYS_ADMIN."
+  {{- end }}
 spec:
+  updateStrategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 1
+      maxSurge: 0
```

Comment:
I think this is default behaviour anyway, adding this just for visibility?

Reply:
i was comparing a template that worked on eks and this was one of the diffs. i was a bit paranoid while testing so i left it. will remove